tests/CMakeLists.txt

```diff
@@ -5,12 +5,14 @@ add_executable(cps-tests
     loader.cpp
     utils.cpp
     version.cpp
+    pc_parser.cpp
 )
 target_link_libraries(cps-tests PRIVATE cps)
 target_link_libraries(cps-tests PRIVATE
     GTest::gtest
     GTest::gtest_main
 )
+target_compile_definitions(cps-tests PRIVATE CPS_TESTS_DIR="${CMAKE_CURRENT_SOURCE_DIR}")
```

Comment:
We should pass this in as a command-line argument or environment variable instead of a preprocessor definition.